R/staggregate.R

```diff
@@ -154,6 +172,28 @@ daily_aggregation <- function(data, overlay_weights, daily_agg){
   return(list(clim_daily, layer_names))
 }
 
+# Function to infer date-times for raster layers based on a time interval
+infer_layer_datetimes <- function(raster_stack, start_date, time_interval) {
```

Comment:
What happens if this is the default NA? Does this part skip if no argument is applied to start_date?

Reply:
You only call infer_layer_datetimes if the start_date is not NA -- if it's the default NA you use the existing layer names